[0.I] Prune and prepare Changelog (#86436)
CleverRaven/Cataclysm-DDA: data/changelog.txt

## Features:

> Toggle to highlight OMTs revealed by maps

Reviewer: ```suggestion
Toggle to highlight areas revealed by using map items
```

[Mods] Non NE region mods have different day/night times dependant on their setting
add a profession condition for EOCs

> Introduces dormant zombies and associated mechanics. They spawn in mass graves in limited numbers for now

Reviewer: ```suggestion
Introduces dormant zombies and associated mechanics
```
Looked it up, it seems #78150 results in enough places having dormant zombies that it's not worth mentioning anything about the location

## Features:
Toggle to highlight OMTs revealed by maps

> Allows dormant zombie types to be defined easily, with a simple new monster flag

Reviewer: We already have a line about dormant zombies and they're a new feature to this release, I don't think this line is needed?

[Mods] Non NE region mods have different day/night times dependant on their setting
add a profession condition for EOCs
Introduces dormant zombies and associated mechanics. They spawn in mass graves in limited numbers for now
Adds treetops and makes trees climbable
Allow bird wings to glide
Allow whitelisting and blacklisting hobbies in scenarios, professions, and globally
Player can eat from camp larder(always). Workers receive spent food from it when being assigned to work (requires NPC needs enabled)
Allow the Chimera thershold trait as a wildcard for most post-threshold traits
You can shatter zombies' torso armour by hitting them real hard

> Unhardcode KI strike / Drunken master

Reviewer: Ki strike / drunken master changes not functional but refactoring so probably not needed

EoCs support mutate_towards directly
make `u_spawn_item` to support the item group whose subtype is `collection`
Add u_wants_to_talk/npc_wants_to_talk EOC effects
Player can eat from camp larder(always). Workers receive spent food from it when being assigned to work (requires NPC needs enabled).
Implement IME pre-edit text for CJK text
Allow effects disabling (conditional) limb flags
Threshold substitutions
Small scale anaerobic digestion system
Shatter a zombie's torso armour with a good hard hit.
Limbify Feathered Arms, rework gaining Bird Wings
Player can eat from camp larder(always). Workers receive spent food from it when being assigned to work (requires NPC needs enabled)
Allow the Chimera thershold trait as a wildcard for most post-threshold traits
You can shatter zombies' torso armour by hitting them real hard
Unhardcode KI strike / Drunken master
Make power grids easier to disconnect, reconnect, and merge parts to
Beast mutants can mutate quills
Adds a new trait, Tiny Stomach
Add new hairstyles from Mawranth's Hair Salon
Unhardcode and limbify attack vectors
Add new function follower_present for use in JSON

> Graphical construction preview

Reviewer: This is a neat new feature, I'd leave it in

> Smartphones, can reveal some of the map around where it was picked-up

Reviewer: ```suggestion
Smartphones can reveal some of the map around where it was picked up
```

Special vision enchantment for Antennae mutation
Add keybind to throw your wielded item
Audio now works on curses mode, not just tiles mode
Add unloading hotkey to AIM menu
Improve blood placement on Riot Damage
Make pulping a resumable activity
Makes sound effects play in slow motion while the player has slowed down time
Underground Temperature now depends on Region Settings
Mods can blacklist mods they are incompatible with
Items can now have faults
Make aggro_character respect anger_relations
River generation overhaul
Faction camp food supply can expire over time
Roads cross streams at culverts
Add Weight Modifying Enchantment
Query to quick-reveal all nearby wires when revealing wall wires
Generic tie-down/haul system for furniture on vehicles
Allow pushing/pulling furniture directly onto and off of vehicles that can load them
Add vehicle enchantments and effects
Toolmods are removable again, and any tool with screw driving quality can remove them
NPCs can handle liquids from crafting
Bashed furniture and terrain produce dust fields
Static NPC factions have their own camps and food stores
Neutral NPCs may now be shoved

> Framework to support Mortar

Reviewer: Was there any gameplay  associated with this added yet? if yes, that'd a be a good thing to add as well.
```suggestion
Framework to support mortar-like long range attacks
```

Add activity to continually bash something until it's destroyed

## Content:
Add conjunctivitis from boomer bile
Add a dentist office map
Add a new appliance, the glassblower's crucible
Adds whispering fungi to fungi zombie mongroup.
Add persistent boomer bile effect that causes a stench and skin irritation
Add new hairstyles from the Mod Mawranth's Hair Salon

> Implement basic saline infusion to help with blood loss

Reviewer: ```suggestion
Implement saline infusion kits to help combat hypovolemic shock
```

Hub now offers armor set printing after HWP is unlocked
Lockets can store photographs
Add diet soda, because screw your free calories!
Addition of many new books, from religious texts to magazines, mainly for fluff purposes
Modularize Desert Eagle, add .50 AE
Addition of three new, big-sized treefrog mutants. Minor tweaks to existing frog mutants

> Further pharmacological expansion infrastructure + adds a few new drugs that make use of the system

Reviewer: Is this the NSAIDs? Did they actually end up doing anything or was that never implemented? IIRC they add vitamins but the vitamins don't do anything or something like that.

Author: This seem to do stuff, no? https://github.com/CleverRaven/Cataclysm-DDA/pull/70239

> JSONize Electrohack Tool

Reviewer: ```suggestion
```
Refactor, no gameplay impact

Add streams to map generation
Added ASCII art for most containers
Adds new jewelry such as cowrie necklaces and various types of hair beads.
Mainlined the innawoods meadow mapgen
Add 7 new frog zombie varities and audit existing frog zombies
Cinnamon for Deluxe Oatmeal
Soldiers' rifles have more/various weapon mods
Reimplement heavy ballistic vest
Adds Chiropteran (bat) mutation line
Adds a subaquatic navigation system CBM
Allow Monster Corpse teeth to be used to make traps and weapons
Adds social/asocial mutations
Add option for natural hair growth over time.
Add earthworms to the game, as a food source
Add cougar kittens and fix cougar taming in mods
Boomers now leave fleshy gibs on detonation
Pictures for House fluff
Added Hub 01 mission signposting LIXA
Crafting Musical Instruments
Additional Achievements
Honk your clown nose!
Add a new LMOE mapgen layout
Eeeeven MOOORE mission context dialogue for Liam (Friends to the End scenario)
Adds a significant amount of extra dialogue and conditionals to Isherwood
Add a new high danger Yrax construct
Add poultry meat
Adds ASCII Art to 619 items

> Created Map Extra that spawns monsters from the Group_Worm

Reviewer: This could probably be reworded to actually tell what it does, something like

```suggestion
More variety for worm-like monsters spawning in the wild
```

Adds Chiropteran (bat) mutation line
Adds a subaquatic navigation system CBM
Allow Monster Corpse teeth to be used to make traps and weapons
Adds social/asocial mutations
Simulate hair growth over time.
Add option for natural hair growth over time.
Add earthworms to the game, as a food source
Add cougar kittens and fix cougar taming in mods
Boomers now leave fleshy gibs on detonation```
Boomers now leave fleshy gibs on detonation
Pictures for House fluff
Added Hub 01 mission signposting LIXA
Crafting Musical Instruments
Additional Achievements
Honk your clown nose!
Add a new LMOE mapgen layout
Eeeeven MOOORE mission context dialogue for Liam (Friends to the End).
Adds a significant amount of extra dialogue and conditionals to Isherwood (Isherwood update part 1, 50% done)
Eeeeven MOOORE mission context dialogue for Liam (Friends to the End scenario)
Adds a significant amount of extra dialogue and conditionals to Isherwood
Add a new high danger Yrax construct
Add poultry meat
Adds ASCII Art to 619
Adds ASCII Art to 619 items
Created Map Extra that spawns monsters from the Group_Worm
Adds a lot of mission-specific and chat/small-talk dialogue to Liam (Friends to the End scenario)

> Rework exoskeletons

Reviewer: I'd leave this in, maybe something like
`Exoskeletons are now modular and have multiple types`

> Adds a new  globally unique special to spawn in the ocean: an Oil Platform !

Reviewer: ```suggestion
Adds a new  globally unique special to spawn in the ocean: an oil platform!
```

Adds still more mission-specific dialogue for Liam (Friends to the End)
Add XL and XS versions for french maid clothes/aprons/hats
Fridges and freezers. What do you mean 'we already have some'?
Add a variety of new fridges and freezers, and update the stats of the old ones
New portal monsters: unviable life forms
Challenge start - hunted forever. Starts with the nemesis, but it comes back stronger everytime it dies
Adds a new  globally unique special to spawn in the ocean : an Oil Platform !
Adds a new  globally unique special to spawn in the ocean: an Oil Platform !
Overhaul and flesh out Barry Isherwood's rescue mission
Prepper Militia Community
Add Snow Cones and Snow Machines
Add three new locations to Aftershock
Add Alcoholic Snow Cones
New jewelry added
Night vision helmet attachments
Add murky water, make regular water safer
Add Alcoholic Snow Cones
10/22 Pistol
Adds a variety of marine plants.
Adds bench vise
Full auto 10/22s

> Reduces craters

Reviewer: `Random crater explosions have been removed from the game`
This is quite significant fix imo so good to mention?

fish emulsion
Static NPC factions have their own camps and food stores
Regen serum
Vitamin Cost per mutation
Adds a new item and recipe to produce fish emulsion from fish, sugars, and water aerobically
Add Serum 48x-"Locutus" (Regeneration serum)
Add Vitamin Cost to mutations
Small lighthouse
Red Phosphorus Using Matchbox Strikers and Potassium Chloride Using Match-heads
Add brick construction to camp expansions
Clean water in open containers reverts to water after a while
Update the basic gas mask, related filter, filter ammunition, and gas mask bag to modern CDDA standards, add possible inserts and outserts, disassembly recipe, and item group goodness.

> Remove the 'Shockstorm' attack from monsters that have it

Reviewer: Also notable gameplay change. Maybe
`Electric zombies no longer have a long range AoE attack.`

> Add game screenshots to data/screenshots

Reviewer: ```suggestion
```
This is just for the flapak packages afaik

> Added a way to fix cyborgs at rubic's castle

Reviewer: ```suggestion
Added a way to fix cyborgs at the Exodii base
```

> Added mashed potatoes.

Reviewer: ```suggestion
Added mashed potatoes
```

Professions & hobbies know gross anatomy & new butchering profs
Slightly Expands Hub-01 and Yrax factions

> Adds a new stationary fungal variant.

Reviewer: ```suggestion
Adds a new stationary fungal variant
```

Adds a new fungus infested mapgen

> Adds new end screen for thresh types, and a unique one.

Reviewer: ```suggestion
Adds new end screen for mutation tresholds, and a unique one
```

> Fungal brute and hulk

Reviewer: ```suggestion
Fungal brute and hulk, more fungalized stages for other zombies
```

Massive expansion of dialogues and messages for various categories

> Add a random NPC class that knows how to fly

Reviewer: ```suggestion
Add a random recruitable NPC class that knows how to pilot helicopters, they may be switched to if player doesn't know how to fly
```

Adds in new .357, .44, and .22 LR lever-action rifles
More carpet colors
Beds are draggable
Large clay pots can now be crafted without a lid and used in the slow sand water filter
Fence posts and bollards are more similar
Remove secret science and military stuff from bank/industry/mall vaults
Added a welding store
Add Vitamin C Supplements
Simple cloth quiver for holding various ammo
Walls now have a chance of having graffiti on them
The walls of most buildings vary more
New scenario: Moonlit Night
Added chokeberries and spicebush

> Add dormant zombies to more locations

Reviewer: We had a line about dormant zombies somewhere at the start, should figure out what kinds of places they can spawn in and mention it there since it's a new feature, and remove this line
```suggestion
```

Add tupelo and nannyberry,  hobblebush and autumn olive trees
New exodii rifle
Remade the 3x3 hospital
Some houses are under construction
Adds car crashes into existing buildings
Basic beach decorations
Add an new plasma gun for sale by the Exodii
Added a small kindergarten
Add Hazelnut Milk, Walnut Milk, and add them to various recipes
Add the menorah and small candles to put in it

> Adds three new lore log snippets for the player to find in smartphones or laptops

Reviewer: I bet there's been way more than three of these in total
```suggestion
More new lore log snippets for the player to find in smartphones or laptops
```

Add an electric rechargeable lighter
Adds marshmallow fluff and fluffernutters
Add radishes and seeds
Riot damage in cities
Another new Exodii Gun: the Khuunäöfaai ordnance cannon, a long range incendiary and chemical weapon.
Adds new rails vehicles
Add in pride/pronoun pins
Add in the guns S&W 617, Ruger PCC, and Marlin Model 60
Adds large wicker baskets

> Dormant zombies map extra

Reviewer: ```suggestion
```
Again dormant zombies is already a headline somewhere earlier

Missile silos are decommissioned
26 and 153 round magazines for the PA md. 68
Added more flooded basements
Added a new 2x1 house
More types of monster spawn in basements
Most toilets now have toilet supplies
Add 2 new social practice recipes
FEMA camps are now mutables
Added a new fast food restaurant with drive-through
Improves/adds missions, rewards and interactions with the New England Community Church
Military outposts have been revamped

> Adds a new nether monster

Reviewer: Which? I'm sure there's been more than one.

New scenario: Moonlit Night
Added chokeberries and spicebush
Add dormant zombies to more locations
Add tupelo and nannyberry,  hobblebush and autumn olive trees
New exodii rifle
Remade the 3x3 hospital
Some houses are under construction
Adds car crashes into existing buildings
Basic beach decorations
Add an new plasma gun for sale by the Exodii
Added a small kindergarten
Add Hazelnut Milk, Walnut Milk, and add them to various recipes
Add the menorah and small candles to put in it
Adds three new lore log snippets for the player to find in smartphones or laptops
Add an electric rechargeable lighter
Adds marshmallow fluff and fluffernutters
Add radishes and seeds
Riot damage in cities
Another new Exodii Gun: the Khuunäöfaai ordnance cannon, a long range incendiary and chemical weapon.
Adds new rails vehicles
Add in pride/pronoun pins
Add in the guns S&W 617, Ruger PCC, and Marlin Model 60
Adds large wicker baskets
Missile silos are decommissioned
26 and 153 round magazines for the PA md. 68
Added more flooded basements
Added a new 2x1 house
More types of monster spawn in basements
Most toilets now have toilet supplies
Add 2 new social practice recipes

> FEMA camps are now mutables

Reviewer: ```suggestion
Larger, more varies FEMA camps
```

Added a new fast food restaurant with drive-through
Improves/adds missions, rewards and interactions with the New England Community Church
Military outposts have been revamped
Adds a new nether monster: Eldritch Huntsman

> Snippetize Tshirt variants

Reviewer: ```suggestion
```

> Exodii Ceramics

Reviewer: ```suggestion
```

Add Arab and Islamic clothes for women
Adds a few other scavengeable options for anvils
Add buildable sturdy wooden shutters for your windows
Add a sunken helicopter location

> Nuclear Power Plant upgrade

Reviewer: ```suggestion
Nuclear power plant location now has an extensive basement
```

> Opiate Vitamin

Reviewer: ```suggestion
```
Refactor, no major gameplay difference

> Add arab and islamic clothes for women

Reviewer: ```suggestion
Add Arab and Islamic clothes for women
```

AIM: when cannot wear, try to wield
Separate keybinding to select default ammo for RELOAD_AND_SHOOT/RELOAD_ONE weapon
Show NEW! in List Items menu
Adding spell name to the in-progress casting widget
Mission UI migrated to imGUI
Provide a visual UI for changing vehicle part shapes

> Display RAS weapon reload time in aim ui

Reviewer: ```suggestion
Display bow reload time in aim UI
```

Warn on setting an auto feature when auto features are off
Show actual real-world distance to mission objective
Add selection menu for cosmetic traits in character creation
Adds message for NPCs completing crafting so it doesn't reference the player incorrectly.
Add the 12-bit rainbow as color palette option

> NPC follower rules can now be viewed and modified in a dedicated, mouse-supported window (imgui)

Reviewer: ```suggestion
NPC follower rules can now be viewed and modified in a dedicated, mouse-supported window
```

Prevent automatically assigning keys to passive bionics
Prevent automatically assigning keys to passive mutations
Zone Manager: "Show All / Hide Distant" highlights selection
Add sorting by amount and value/volume in advanced inventory search
Addiction withdrawals are a toggleable distraction
Select CC-Sounds as default soundpack if it's present on first run
Migrate uilist implementation to use ImGui
Allow ebooks in read command

> Make ImTui render text as text, so that the players can read our spiffy new menus

Reviewer: ```suggestion
```
Just a bugfix

Toggle to only draw overmap during autotravel
Mouse input in uilists does not block keyboard navigation

> Add imgui version of the iteminfo window

Reviewer: Iteminfo window migrated to ImGui

Updated default keybindings for changing start dates in character creation menu
Rewrite the loading screen to use ImGui
Custom loading screen the game deserves
Wrap the descriptions in the zone menu so that it doesn’t have a ridiculous width
Map UI: hint custom keybind for `CHOOSE_DESTINATION`
Only list non-zero damage types on melee weapons
Misc. improvements to NPC goto-location options
Repeat survey missions without leaving the interface

> Add daily health widget

Reviewer: ```suggestion
Add a daily health sidebar widget
```

Add labels to structured sidebar tiny bodygraphs
Use Freetype to render fonts for ImGui, and use multiple fonts in the ImGui interface

> Wrap the text in the menu for choosing your new faction camp type

Reviewer: ```suggestion
```
Imgui bugfix

Missions track when they were completed/failed
Option to set all of a companion's job priorities at once

> The mission UI can scroll to read longer description

Reviewer: ```suggestion
```
Imgui bugfix

Added information to crafting menu about crafting speed modifier gained by mutations

> Improved text wrapping in spellbook and spellcasting menus

Reviewer: ```suggestion
```
Imgui bugfix

Camp crafting uses the normal crafting GUI
Display the power draw of battery chargers

> Increased available space for skills chapter in description tab of character creation menu

Reviewer: ```suggestion
```

> ImGui inherits theme colors

Reviewer: ```suggestion
```
Imgui is a new feature in this release and this is how it's expected to work

Add sorting by price to weight ratio
Filter items by can_contain `L:`, `V:`, `M:` (longest_side, volume, mass); affects AIM, inventory…
Migrate compare item menu to imgui
Better skill requirement UI for martial arts

> Scale spell window to spell list and terminal size

Reviewer: ```suggestion
```

> Fix compact aim window

Reviewer: ```suggestion
```

> Fix NPC rules window scaling for small terminal sizes

Reviewer: ```suggestion
```

Add history to Construction Search
Load all of the user’s chosen font files as fallbacks for ImGui to use
Extended description window (x->e) shows potential deconstruction and bash yields
Focus the default selection in query popups
Zone manager will expand to match a wider sidebar.

> Add numpad keybinds for panning uilists

Reviewer: ```suggestion
```

Add options to sort item list by item name
Dont show liquids from held or worn container
Move camp-specific recipes to their own category
Add option to disable mouse

> Add "in use" info to item's name and info if item has "activity_var" var.

Reviewer: ```suggestion
Add "in use" info to item's name if the item has been reserved by an NPC for camp work
```

> Create a terminal-like default theme for Imgui

Reviewer: ```suggestion
```

Make spell difficulty accessible in math
Add max_worn field, migrate ONLY_ONE flag
Upgrade clang-tidy to LLVM 18
Unified basement palettes
Improve error reporting
Refactor wield
EOCs for vehicles
Introduces 'light_multiplier' weather_type.json property
Make AIM actions testable
Allow setting 0 spoil time to enable copy-from for 0 spoiling
Warn on load if files have been modified
Always Display Spell Tabs
Allow monsters to have different climb/swim/dig speeds
Mapgen: removed unused, obsoleted and pointless JSON fields

> Zstandard compressed save data for disk space savings.

Reviewer 2: ```suggestion
Implement compressed saves
```
